fetch-validator-status/networks.json

```diff
@@ -50,5 +50,9 @@
   "itn": {
     "name": "Indicio TestNet",
     "genesisUrl": "https://raw.githubusercontent.com/Indicio-tech/indicio-network/main/genesis_files/pool_transactions_testnet_genesis"
+  },
```

Comment:
The `main` branch has updated the network list.  I'll be rebasing the `monitoring-stack` branch on it.